contributing/contributing-code.md

```diff
@@ -90,8 +100,10 @@ Once you have found a reviewer, make sure to @-mention them on any issues / PRs
   - [Push your changes](./getting-started-e2e.md#push-your-changes-to-your-github-fork)
   - [Create a Pull Request (PR)](./getting-started-e2e.md#send-a-pull-request-ie-request-a-code-review).
   - Make sure the presubmit checks shown on your PR on GitHub passes (e.g. no lint and type check errors, tests are passing).
+  - Add reviewers to your PR that will fulfill the requirements of code review and approval documented in the [Code review and approval](#code-review-and-approval) section.  (Your reviewer can help with this.)
   - [Respond to feedback](./getting-started-e2e.md#respond-to-pull-request-comments).
-- After your PR is approved, it will merged by your reviewer or one of the other approvers. To check on your changes and find out when they get into production, read [See your changes in production](./getting-started-quick.md#see-your-changes-in-production).
+- After your PR is approved, it will merged by one of the Reviewers/Collaborators who reviewed your code.
```

Comment:
Couldn't we use the same language here as below, e.g.: "Once the PR has been approved, anyone with commit rights to the repository may merge the PR, including its author if they have commit rights."

Reply:
Since this is from the perspective of someone going through the review process I wanted to make it a bit more actionable--i.e. if the code is not merged what should they do?  I've updated it to make this more obvious.